specification/metadata/providerprefix.md

```diff
@@ -0,0 +1,17 @@
+# Provider Prefix
+
+Used as the prefix for the name of custom columns, useful for the consumer of the data to quickly identify custom column extensions to the FOCUS schema implemented by the provider.
```

Comment:
Conceptually, the provider prefix belongs to the provider, not to the data generator: two data generators for the same provider should have the same prefix.
Maybe we should not go as far as splitting provider metadata and generator metadata, but maybe the specification could say that the provider prefix must be the same for all data generators addressing the same provider.